### ISSUE TITLE
Crash on install....... then unable to run ./src/setup.sh after rebooted

### ISSUE THREAD

**Issue by hraley** [nicholasadamou/Raspberry-Pi-Auto-WiFi-Hotspot-Switch-Internet#3]
Hey there!

Great script, I found your Git after reading this article
https://www.raspberryconnect.com/projects/65-raspberrypi-hotspot-accesspoints/158-raspberry-pi-auto-wifi-hotspot-switch-direct-connection

What you have created looks very useful, thanks!! 😄 

My set up:
- Pi Zero W
- Completely fresh install of Raspbian Buster Lite, then updated
- Configured to connect to my home network
- SSH enabled
- Connection with Putty established
- Installed Git

After cloning your git, I ran ` ./src/setup.sh` It seemed to work, then hung on 97% of something, i can't remember what.
Ended up hard rebooting (power disconnect etc)
SSH back in, found the repository and attempted to run ` ./src/setup.sh` again...... however this time it freezes on
![image](https://user-images.githubusercontent.com/6191853/78473512-16f1ad80-7739-11ea-9547-acffe68b6e23.png)

Is there any way I can uninstall and restart?
Or manual get around the hang when trying to disable `hostapad`

Thanks a lot 👍

**Comment by nicholasadamou** [comment]
Hi! Thank you for your issue and your kind words! I will be looking into the problem with more detail today. 

As for uninstalling the script, I have added a program that can be used to remove all programs and files created/installed when using the setup script. It can be invoked via:

```bash
cd Raspberry-Pi-Auto-WiFi-Hotspot-Switch-Internet/
bash src/uninstall.sh
```

Don't forget to `git pull origin master` to get the latest version of this repository.

**Comment by hraley** [comment]
I wiped the sd card and did a complete fresh install...... 🙄 🙄 
Same issue. Got through a lot of stuff, installed both `hostapd` and `dnsmasq-base`

Here is a screenshot
![image](https://user-images.githubusercontent.com/6191853/78499357-23c9cd80-7748-11ea-9226-44538d323791.png)

Thanks for the speedy reply, greetings from 🌞  Yorkshire England!!!

**Comment by hraley** [comment]
Upon second thoughts, I'm only really after the functionality described here
https://www.raspberryconnect.com/projects/65-raspberrypi-hotspot-accesspoints/158-raspberry-pi-auto-wifi-hotspot-switch-direct-connection

I don't actually need internet pass through when the Pi is in AP mode.... just a local connection when out of range of usual WiFi. Maybe you script is a little overkill for me.

The reason i liked what you'd done is i do need to implement this on quite a few Pi's - hence doing some quick testing on a fresh instillation before i go ahead and sort the rest out.

Thnx

**Comment by nicholasadamou** [comment]
I understand what you are saying. I created this repository basically for the same reason, but I wrote this script to mimic the guide found here: [raspberry-pi-auto-wifi-hotspot-switch-internet](http://www.raspberryconnect.com/network/item/330-raspberry-pi-auto-wifi-hotspot-switch-internet). Which means, you are right. This script would be overkill for the tasks mentioned in [raspberry-pi-auto-wifi-hotspot-switch-direct-connection](https://www.raspberryconnect.com/projects/65-raspberrypi-hotspot-accesspoints/158-raspberry-pi-auto-wifi-hotspot-switch-direct-connection).
